action.yml

```diff
@@ -1,26 +1,58 @@
-name: "Lazy Action Python"
-description: "Github action to perform build, test , scan and generate image."
+---
+# yamllint disable rule:line-length
+name: Actions Python
+description: >
+  Github action to perform build, test , scan and generate image.
 inputs:
   dockerfile_dir_path:
-    description: "Directory path to the dockerfile"
+    description: Directory path to the dockerfile
     required: false
-    default: "."
+    default: .
   ecr_repository:
-    description: "ECR repository name"
+    description: ECR repository name
     required: true
   container_push_enabled:
-    description: "Enable Build and Push Container Image"
+    description: Enable Build and Push Container Image
     required: true
-    default: "true"
-  test_framework:
-    description: "Framework for Tests"
+    default: 'true'
+  python-version:
+    description: >
+      The python-version input is optional.
+      If not supplied, the action will try to resolve the
+      version from the default `.python-version` file.
+      If the `.python-version` file doesn't exist Python or PyPy
+      version from the PATH will be used.
+      The default version of Python or PyPy in PATH varies
+      between runners and can be changed unexpectedly
+      so we recommend always setting Python version explicitly
+      using the python-version or python-version-file inputs.
```

Comment:
should remove the reference for the python-version-file input because it's a separate input in actions/setup-python